Assets/MobileDevices.SDK/README.md

```diff
@@ -0,0 +1,37 @@
+# Movbile Devices SDK (Android / iPhone)
```

Comment:
We should consider in which folder these components belong. Quite possibly many should be folded into the MixedRealityToolkit.* folders.